lib/parallel_tests.rb

```diff
@@ -6,28 +6,32 @@
 module ParallelTests
   WINDOWS = (RbConfig::CONFIG['host_os'] =~ /cygwin|mswin|mingw|bccwin|wince|emx/)
   RUBY_BINARY = File.join(RbConfig::CONFIG['bindir'], RbConfig::CONFIG['ruby_install_name'])
-  DEFAULT_MULTIPLE = 1.0
+  DEFAULT_MULTIPLY_PROCESSES = 1.0
 
   autoload :CLI, "parallel_tests/cli"
   autoload :VERSION, "parallel_tests/version"
   autoload :Grouper, "parallel_tests/grouper"
   autoload :Pids, "parallel_tests/pids"
 
   class << self
-    def determine_number_of_processes(count)
```

Comment:
This renaming broke bundler + ruby specs. Any chance to get it back as an alias or should we adapt?

Reply:
will re-rename, thx!

Reply:
4.10.1

Reply:
Thanks!